### PR TITLE
Support  `global`,`local`, `on commit` for `create temporary table`

### PR DESCRIPTION
**Pull request by gandronchik**
CREATE [GLOBAL | LOCAL] TEMPORARY TABLE table (col INT) ON COMMIT [DELETE ROWS | DROP | PRESERVE ROWS]

add support GLOBAL | LOCAL
add support ON COMMIT [DELETE ROWS | DROP | PRESERVE ROWS]

### CODE REVIEW

File: src/ast/mod.rs [hunk 1]
```diff
@@ -1276,9 +1280,18 @@ impl fmt::Display for Statement {
...
                     external = if *external { "EXTERNAL " } else { "" },
+                    global = if global.is_some() {
+                        if global.unwrap() == true {
+                            "GLOBAL "
+                        } else {
+                            "LOCAL "
+                        }
+                    } else {
+                        ""
+                    },
```
Comment (alamb) [inline src/ast/mod.rs:1294]: I think more idomatic rust would be the following (it avoids `unwrap` when you know it is already `Some`)

```suggestion
                    global = global.map(|global| {
                        if global.unwrap() == true {
                            "GLOBAL "
                        } else {
                            "LOCAL "
                        }
                    }).unwrap_or(""),
```

**Review by alamb — APPROVED** [review]
I had one tiny style suggestion, but otherwise this looks great 👍

**Review by alamb — APPROVED** [review]
Thanks @gandronchik

**Comment by alamb** [comment]
Seems like there is a clippy failure @gandronchik

**Comment by gandronchik** [comment]
> 



> Seems like there is a clippy failure @gandronchik

Oh, god, sorry) fixed

**Comment by alamb** [comment]
Thanks again @gandronchik  👍

**Pull request merged**